core/src/program.rs

```diff
@@ -98,14 +98,10 @@ impl Program {
     }
 
     /// Set allocations as a set of page numbers.
+    // TODO: used in gtest only. Remove after gtest is removed
```

Comment:
It would be nice to create corresponding issue and mention it there

Reply:
`gtest` won't be removed, but open an issue cause such functionality will disappear from there